.slim-lint.yml

```diff
@@ -0,0 +1,7 @@
+exclude:
+  - 'vendor/bundle/**/*.slim'
+  - 'app/views/shared/_checkbox_group.html.slim'
```

Comment:
Curious about why this one is so smelly 😂

Reply:
The problem there is:

```
app/views/shared/_checkbox_group.html.slim:11 [W] RuboCop: Lint/UselessAssignment: Useless assignment to variable - `hint_method`. (https://rubystyle.guide#underscore-unused-vars)
```
I could not manage to disable that specific check only for that file. It would be nice not to assign that variable that is used in another file